src/api/getWilayaByPhoneCode.js

```diff
@@ -9,15 +9,18 @@ const getWilayaByPhoneCode = data =>
 * //returns {mattricule: '6', name: 'Béjaïa', ...} because the phone code 34 belongs to Béjaïa
 * getWilayaByPhoneCode(34)
 *
-* @param { Number | String } phoneCode is a number if the latter corresponds to the wilaya's phone code and a string if's the full phone number
+* @param { Number | String } phoneCode the phone code
 * @param {String[]} projection a list of wilaya object attributes to keep
 * @returns { Object | undefined } Returns the target object, or undefined
 */
   (phoneCode, projection) => {
-    const phoneCodeVar =  typeof phoneCode === "number"
-      ? phoneCode
-      : parseInt(phoneCode.substring(1, 3), 10);
-    const wilaya = data.find(w => w.phoneCodes.includes(phoneCodeVar));
+    if(Number.isNaN(phoneCode)){
+      return;
+    }
+    const parsedPhoneCode = Number.parseInt(phoneCode, 10);
+    const wilaya = data.find(w => w.phoneCodes.includes(parsedPhoneCode));
```

Comment:
Running `npm run test.mutation` returned that we didn't cover this line 😞 .

![image](https://user-images.githubusercontent.com/9090674/94991206-cf31ce80-0581-11eb-8d60-cd46b068fa09.png)

You can [put the blame on me](https://youtu.be/26WBT1ZdLdc?t=88) 😐 . 

Anyways, let's do this and I promise it will work:

```suggestion
    const parsedPhoneCode = Number.parseInt(phoneCode, 10);

    if(Number.isNaN(parsedPhoneCode)){
      return;
    }
     
     const wilaya = data.find(w => w.phoneCodes.includes(parsedPhoneCode));
```